0000-template.md

```diff
@@ -0,0 +1,116 @@
+# Table of Contents
+- [Summary]
+- [Tracking Issues](#tracking-issue)
+  - [Discourse Discussion](#discourse-discussion)
+- [Motivation]
+- [Guide Level Explanation](#guide-level-explanation)
+- [Reference Level Explanation](#reference-level-explanation)
+- [Drawbacks]
+- [Rationale and Alternatives](#rationale-and-alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+
+# Basic Info
+[basic]: #basic-info
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- [Tracking Issue](#tracking-issue): (leave this empty)
+- [Discourse Link](#discourse-discussion): (if there is one)
+
+After filling out these fields, please proceed to fill out the remaining sections of this document.
+
+# Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+# Tracking Issue
+[tracking-issue]: #tracking-issue
+<details>
+<summary>Info about Tracking Issues and how they work</summary>
+These are issues created on GitHub that track the implementation progress of the RFC. They link to related issues, have checkboxes, or other structures that make them more of a meta-issue than an issue for adding a specific feature or fixing a bug.
```

Comment:
Could you clarify here if this is something you're supposed to create when you file the RFC or how the process in general goes? I don't quite feel sure about it myself just reading this and not knowing much about the Rust RFC process.

Reply:
Actually I think this should go in the README, not the template

Reply:
I moved it and added that the sub-team would setup the tracking issue.